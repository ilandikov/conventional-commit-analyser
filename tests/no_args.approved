Error: Please provide at least one repository path using --path.
Usage: ./conventional-commit-analyser.sh --path <path1> [--path <path2> ...] [--author <author>] [--show-skipped-commits] [--by <period>] [--commit-days] [--risk]
