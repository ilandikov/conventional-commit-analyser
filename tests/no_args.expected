Error: Please provide exactly one repository as an argument.
Usage: ./conventional-commit-analyser.sh <repository>
